rest/perf_routes.go

```diff
@@ -283,3 +283,41 @@ func (h *perfGetChildrenHandler) Run(ctx context.Context) gimlet.Responder {
 	}
 	return gimlet.NewJSONResponse(perfResults)
 }
+
+///////////////////////////////////////////////////////////////////////////////
+//
+// POST /perf/signal_processing/recalculate
```

Comment:
While I think having a manual switch to recalculate everything is good, there should be a background process to add the backfil on some interval. The application has all the information it needs to be figure out what and when to recalculate without requiring an external process to pull this lever.

Reply:
Added a tix for a cron based backstop https://jira.mongodb.org/browse/MAKE-1069